integration-tests/executor.go

```diff
@@ -70,6 +70,15 @@ func ReadBoolEnvVar(env string) bool {
 	return e
 }
 
+func GetQATag(base_tag string) string {
```

Comment:
[nit] this might be more appropriately placed in the utils.go file

Reply:
I had a pretty hard time deciding where to place this function and somehow completely missed `utils.go`, I'll move it there.